gray/processing.py

```diff
@@ -11,34 +14,102 @@
 log = logging.getLogger(__name__)
 
 
-class FormattingError(Exception):
+class Error(Exception):
```

Comment:
`GrayError`, maybe? `Error` is a too broad name.

Reply:
Done